bundles/binding/org.openhab.binding.zwave/database/aeon/zw096.xml

```diff
@@ -191,7 +191,6 @@
             <Type>byte</Type>
             <Default>DDA0DD</Default>
```

Comment:
This value should be in decimal. I know you didn't add it, but do you mind fixing it?